.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            max-parallel: 1
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                include:
+                    - description: 'Symfony 3.4'
```

Comment:
Wrong description. this job is testing lowest deps, which is not the same than a job testing symfony 3.4 (which would rather check an uptodate 3.4 version)